cmake/Codegen.cmake

```diff
@@ -110,6 +110,12 @@ if(INTERN_BUILD_ATEN_OPS)
     endif(MSVC)
   endif(CXX_AVX2_FOUND)
 
+  if(CXX_VSX_FOUND)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DHAVE_VSX_CPU_DEFINITION")
+    LIST(APPEND CPU_CAPABILITY_NAMES "VSX")
+    LIST(APPEND CPU_CAPABILITY_FLAGS "${OPT_FLAG}  ${CXX_VSX_FLAGS}") 
+  endif(CXX_VSX_FOUND)
+
```

Comment:
I don't think AVX and VSX were supposed to be found at the same time, and the actual preprocessing code doesn't handle this case. So perhaps adding the following:

```cmake
if (CXX_VSX_FOUND and CXX_AVX_FOUND)
    message(FATAL_ERROR "Both AVX and VSX are present!")
endif()
```